### ISSUE TITLE
config fails on latest caddy

### ISSUE THREAD

**Issue by gedw99** [infogulch/todos#1]
I am using github.com/caddyserver/caddy/v2/cmd/caddy latest with template compiled in.

but I get...

´´´sh
cd todos && caddy-cmd run --config Caddyfile
2024/04/29 03:49:55.281 INFO    using provided configuration    {"config_file": "Caddyfile", "config_adapter": ""}
Error: adapting config using caddyfile: parsing caddyfile tokens for 'route': unrecognized directive: file_server - are you sure your Caddyfile structure (nesting and braces) is correct?, at Caddyfile:9
```

I think this changed with latest caddy.

**Comment by gedw99** [comment]
its my fault 

```sh
cd todos && caddy-cmd list-modules
admin.api.load
caddy.config_loaders.http
caddy.logging.writers.discard
caddy.logging.writers.stderr
caddy.logging.writers.stdout

  Standard modules: 5

  Non-standard modules: 0

  Unknown modules: 0
```

**Comment by infogulch** [comment]
You'll want to import xtemplate-caddy like

```go
import _ "github.com/infogulch/xtemplate-caddy"
```

**Comment by infogulch** [comment]
Yes well I haven't updated xtemplate-caddy to reference the new xtemplate config yet, that's probably what needs fixed first. There may be other issues as well.

**Comment by gedw99** [comment]
I think that’s it.

I am on mobile at the moment and can’t copy paste the 8 or some compile errors ..

the main.go is very simple. If you want it let me know.
